File: desafio_backend_stefanini/desafio_backend_stefanini/ClientApp/src/app/types/Pessoa.ts
import { Cidade } from "./Cidade";

export type Pessoa = {
  id: string,
  nome: string,
  cpf: string,
  cidadeId: string,
  cidade: Cidade
};

export type PessoaIncluir = {
  nome: string,
  cpf: string,
  cidadeId: string
};

export type PessoaAlterar = {
  id: string,
  nome: string,
  cpf: string,
  cidadeId: string
}

File: desafio_backend_stefanini/TesteAPI/Models/Pessoa.cs
using desafio_backend_stefanini.API.Interfaces;

namespace desafio_backend_stefanini.API.Models
{
    public class Pessoa : IEntity
    {
        public Guid Id { get; set; } = Guid.NewGuid();
        public string Nome { get; set; }
        public string Cpf { get; set; }


        public Guid CidadeId { get; set; }
        public Cidade Cidade { get; set; }

    }
}
